### PR TITLE
company link GA and fixes for missing event O_O

### PR DESCRIPTION
**Pull request by Pomax**
Addresses https://github.com/mozilla/foundation.mozilla.org/issues/2024

Adds company link (seen as product price link on the page) and update article links (L8 and L9 in https://docs.google.com/spreadsheets/d/1x-CP_HYaAs3oRv2cjbSFu8w5Z938E4cyHQ3hvh17c6c/edit#gid=980429434)

Also fixes a missing `id` for the "share link" button, which had fallen off the GA map.

### CODE REVIEW

**Comment by Pomax** [comment]
we're not, I'm blaming rebase, which given how many times this has been rebased is not unreasonable >_>

**Comment by mmmavis** [comment]
Cool. That makes sense. Thought those 2 files were just removed and the code never got moved to another file.

File: source/js/buyers-guide/product-analytics.js [hunk 1]
```diff
@@ -1,117 +1,128 @@
...
-    'reading-level-link': {
+    '#reading-level-link': {
       category: `product`,
       action: `carnegie mellon reading level links`,
       label: `reading level link for ${productName}`,
       transport: `beacon`
+    },
+
+    // product updates
+    'a.product-update-link': {
```
Comment (mmmavis) [inline source/js/buyers-guide/product-analytics.js]: one suggestion:

we can probably just have `.product-update-link` here?

Comment (Pomax) [reply source/js/buyers-guide/product-analytics.js]: yep

**Pull request merged**